ci/before-deploy.sh

```diff
@@ -50,3 +55,7 @@ PKG_NAME_VER="python-gssapi-${PYTHON_GSSAPI_VERSION}"
 
 tar -czvf ./tag_build/${PKG_NAME_VER}.tar.gz --exclude='dist' --exclude='tag_build' --exclude='.git' --exclude='travis_docs_build' --exclude='.git' --transform "s,^\.,${PKG_NAME_VER}," .
 sha512sum --binary ./tag_build/${PKG_NAME_VER}.tar.gz > ./tag_build/${PKG_NAME_VER}.sha512sum
+
+# Rename and checksum the wheel
+mv dist/*.whl dist/"${PKG_NAME_VER}.whl"
```

Comment:
This produces a file named: python-gssapi-1.6.5.whl.  Looking at [PEP 427](https://www.python.org/dev/peps/pep-0427/#file-name-convention), I'm not convinced this is right.

More generally, we have C code.  I believe this means we need to do for Linux what we do for Windows and build three times.  I'm not sure if it's best to add two or three more jobs for that.

Reply:
If you would prefer me to just do this myself instead, let me know :)

Reply:
Right, the name is incorrect, should be underscores instead of dashes. Which begs the question why don't we use the correct name in setup.py and sdist/bdist will produce the right one?

Reply:
> More generally, we have C code. I believe this means we need to do for Linux what we do for Windows and build three times.

I have not explored that in details, I just saw a `python setup.py bdist_wheel` and gave it a go. What do you mean by "build 3 times" ?

Reply:
Well, you did the correct build, but then you renamed it :)

Take a look at the Windows wheels here: https://pypi.org/project/gssapi/#files

There are three of them - and they're specific to python version, which is what I was trying to highlight by linking the PEP.  If you look at the Windows wheel building (in release.yml), there are in fact three separate jobs - one for each python version we build for.

I'm not aware of a way to make one build work for multiple python versions, so my inclination would be to do one for each of the three we support, as we do for Windows.  Does that make sense?

Reply:
> Well, you did the correct build, but then you renamed it :)

I got confused by the rename in the windows release script. 

For the multiple python versions: they can be specified in the job matrix but I think before-deploy.sh runs inside a docker container so not sure how to specify different python version in there.

Reply:
Would a `dnf install pythonX.Y` inside the Fedora container suffice ?